flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/JoinITCase.scala

```diff
@@ -1140,48 +1140,4 @@ class JoinITCase(state: StateBackendMode) extends StreamingWithStateTestBase(sta
     val expected = Seq("Hi,Hallo", "Hello,Hallo Welt", "Hello world,Hallo Welt")
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
-
-  @Test
-  def testJoinOnChangelogSource(): Unit = {
```

Comment:
why this test is removed ?

Reply:
This has been moved to `ChangelogSourceITCase#testRegularJoin` to reuse the tests.